data/json/items/armor/hats.json

```diff
@@ -298,7 +298,6 @@
     "description": "A stylish hat made of faux fur.  Like real fur, but without the suffering, if the tag is to be believed.  Very warm.",
     "material": [ "faux_fur", "cotton" ],
     "covers": [ "head" ],
-    "flags": [ "FANCY" ],
```

Comment:
I'd put this back and add FANCY to hat_fur, particularly given the price tag indicating what sort of fur/faux_fur hat we're talking about.

Reply:
These are outerwear, and stumble across the "no practical clothing" guideline, at least situationally.